src/status_im/multiaccounts/biometric/core.cljs

```diff
@@ -81,6 +82,7 @@
       (.catch #(callback nil))))
 
 (defn get-supported [callback]
+  (log/debug "[biometric] get-supported")
```

Comment:
do we need this logs? (I don't mind keeping them, just wondering if they were leftovers)

Reply:
I can remove this one, but they are actually useful during debugging, sometimes it's a bit hard to figure what's hidden behind the chain of re-frame's events.

Reply:
no that's fine,  I was just wondering if they were leftovers, by all means keep them if useful